packages/terra-tag/translations/en-CA.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Terra.tags.label.delete": "Remove {tagLabelName}",
+  "Terra.tags.label.rollupTag": "{tagsNotVisibleCount} more...",
+  "Terra.tags.label.showLess": "Show Less",
+  "Terra.tags.hint.removable": "item is removable via Delete or Back Space key",
+  "Terra.tags.hint.selectable": "for additional information, button is selectable via Space Bar or Enter key",
+  "Terra.tags.hint.selectableAndRemovable": "item is removable via Delete or Back Space key, for additional information button is selectable via Space Bar or Enter key",
```

Comment:
It should be a period (.) after "Back space key", not a comma.

Reply:
Changes Made. Thanks